### PR TITLE
remove as XXXX of TypeScript (file: google_news_result.ts)

### PR DESCRIPTION
**Pull request by vibh1103**
Fix #203

### CODE REVIEW

File: apps/block/google_news_result.ts [hunk 1]
```diff
@@ -67,7 +67,7 @@ export class GoogleNewsResult extends SearchResultToBlock {
     );
     this.valid = true;
     this._canRetry = true;
-    this.url = href;
+    this.url = href !== null ? href : "";
```
Comment (ikemo3) [inline apps/block/google_news_result.ts:70]: @vibh1103 

If href is null, the url will be empty and should not be blocked.
Please refer to the previous titleElement code(L50).

Comment (vibh1103) [reply apps/block/google_news_result.ts:70]: Ok

Comment (vibh1103) [reply apps/block/google_news_result.ts]: Done

File: apps/block/google_news_result.ts [hunk 2]
```diff
@@ -67,7 +67,7 @@ export class GoogleNewsResult extends SearchResultToBlock {
     );
     this.valid = true;
     this._canRetry = true;
-    this.url = href;
+    this.url = href ? href : "";
```
Comment (ikemo3) [inline apps/block/google_news_result.ts:70]: This won't work. Cannot block when href is null/empty/undefined, so `this.valid = false` and `this._canRetry = false`. There is a code in L50 for when titleElement === null, so please do the same thing there.

**Review by ikemo3 — CHANGES_REQUESTED** [review]
@vibh1103 

Please check the comments and make corrections.

**Comment by vibh1103** [comment]
@ikemo3 Changes done please review again

File: apps/block/google_news_result.ts [hunk 3]
```diff
@@ -34,10 +34,10 @@ export class GoogleNewsResult extends SearchResultToBlock {
       return;
     }
 
-    let href = anchor.getAttribute("href") as string;
+    let href = anchor.getAttribute("href");
```
Comment (ikemo3) [inline apps/block/google_news_result.ts:37]: I realized here that an early return would be sufficient.
However, there's no issue with this change, so I'll approve it.

**Review by ikemo3 — APPROVED** [review]
LGTM!

**Pull request merged**